tests/unit/test_refresh_utils.py

```diff
@@ -226,7 +225,11 @@ def test_seconds_until_refresh_over_1_hour() -> None:
     # using pytest.approx since sometimes can be off by a second
     assert (
         pytest.approx(
-            _seconds_until_refresh(datetime.utcnow() + timedelta(minutes=62)), 1
+            _seconds_until_refresh(
+                datetime.datetime.now(datetime.timezone.utc)
+                + datetime.timedelta(minutes=62)
```

Comment:
Same here, but with 62. What is 62?

Reply:
this is explained in the docstring of the test... testing our scheduling logic for cert expiring in over an hour
```python
"""
Test _seconds_until_refresh returns proper time in seconds.
    If expiration is over 1 hour, should return duration/2.
"""
```